Sources/Commands/SwiftTestTool.swift

```diff
@@ -390,7 +390,7 @@ public struct SwiftTestTool: SwiftCommand {
         let ranSuccessfully = runner.test(outputHandler: {
             // command's result output goes on stdout
             // ie "swift test" should output to stdout
-            print($0)
+            print($0, terminator: "")
```

Comment:
There are some other necessary changes that are missing. Look for other uses of `test(outputHandler` in the repo.

Reply:
https://github.com/apple/swift-package-manager/pull/7126/commits/37ec933ec6cd8daea9849567d9512a1b2ea21e3a

Reply:
@grynspan ^^

Reply:
Woohoo!